src/sentry/static/sentry/app/components/charts/percentageBarChart.jsx

```diff
@@ -0,0 +1,131 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import BarSeries from './series/barSeries.jsx';
+import BaseChart from './baseChart';
+import Tooltip from './components/tooltip';
+import XAxis from './components/xAxis';
+import YAxis from './components/yAxis';
+
+const FILLER_NAME = '__filler';
+
+/**
+ * A stacked 100% column chart over time
+ *
+ * See https://exceljet.net/chart-type/100-stacked-bar-chart
+ */
+export default class PercentageBarChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+
+    series: PropTypes.arrayOf(
```

Comment:
It's almost (but not exactly) the same as Series/Series unit prop type in https://github.com/getsentry/sentry/pull/9248 which also allows data to be a list of numbers cc/ @ayesha-omarali